content-src/asrouter/schemas/message-format.md

```diff
@@ -19,6 +22,11 @@ Field name | Type     | Required | Description | Example / Note
   content: {
     title: "Find it faster",
     body: "Access all of your favorite search engines with a click. Search the whole Web or just one website from the search box."
+  },
+  targeting: "hasFxAccount && !addonsInfo.addons['activity-stream@mozilla.org']",
+  caps: {
```

Comment:
let's call this `frequency` for clarity since you have a `cap` already defined in `custom`